apps/web/pages/getting-started.tsx

```diff
@@ -259,6 +253,44 @@ export default function Onboarding(props: inferSSRProps<typeof getServerSideProp
     token: string;
   }>({ resolver: zodResolver(schema), mode: "onSubmit" });
 
+  const fetchUsername = async (username: string) => {
+    const response = await fetch(`${process.env.NEXT_PUBLIC_APP_URL}/api/username`, {
```

Comment:
@zomars Just checking out if this value it's the right for the job?

Reply:
Yup